provision-rspamd.sh

```diff
@@ -12,6 +12,19 @@ install_rspamd()
 	stage_pkg_install rspamd || exit
 }
 
+configure_redis()
+{
+	if ! zfs_filesystem_exists "$ZFS_DATA_VOL/redis"; then
+		return
+	fi
+
+	tell_status "add Redis address, for default Lua modules backend"
+	echo "redis {
+	servers = \"$(get_jail_ip redis):6379\";
+	db    = \"5\";
+}"  | tee -a "$_etc/rspamd/rspamd.conf"
+}
+
```

Comment:
rather than echo to a pipe to tee, its generally a better practice to invoke tee with a heredoc:
```sh
	tee -a "$_etc/rspamd/rspamd.conf" <<EO_REDIS
	redis {
		servers = "$(get_jail_ip redis):6379";
 		db    = "5";
	}
EO_REDIS
```
Yes, I know, I don't always do that. My lame excuse is, it's usually a one line echo statement that grew, and I haven't refactored it yet. We're about to see why we should..